static/scss/_variables.scss

```diff
@@ -53,6 +53,7 @@ $white-smoke: #f5f5f5;
 $trolley-gray: #808080;
 $dodger-blue: #0084ff;
 $night-rider: #303030;
+$white-smoke-dark: #f0f0f0;
```

Comment:
We have a definition for `$white-smoke: #f1f1f1;`, which was overwritten by a new definition for `$white-smoke: #f5f5f5;`. Can you delete the first `$white-smoke` definition, then move this to be defined directly under the second `$white-smoke` definition?

Reply:
sure